src/react-disposable-decorator.js

```diff
@@ -8,7 +8,7 @@ export default function(DecoratedComponent) {
   }
 
   return class Disposable extends React.Component {
-    static displayName = `Disposable(${DecoratedComponent.displayName || DecoratedComponent.name})`
+    static displayName = DecoratedComponent.displayName || DecoratedComponent.name
```

Comment:
This might make the react dev tools more confusing because there will be two components with the same name. But it makes testing a lot simpler with enzyme because snapshots aren't messed with and `wrapper.find('CompName')` works.

Reply:
Hm, interesting tradeoff. Do you think it's worth it?